### PR TITLE
Feature/update collection

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-components/components/Button/PrimaryButton.re [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[@bs.module "./PrimaryButton"]
```
Comment (davodesign84) [inline packages/kauri-components/components/Button/PrimaryButton.re]: Sigh, creating more components in .re? :(

File: packages/kauri-components/components/Headers/CollectionHeader.re [hunk 2]
```diff
@@ -31,20 +31,24 @@ module Styles = {
         flex(1),
         alignItems(`center),
```
Comment (davodesign84) [inline packages/kauri-components/components/Headers/CollectionHeader.re]: Would be good to start porting back to JS

File: packages/kauri-web/components/containers/CreateCollectionForm/index.js [hunk 3]
```diff
@@ -50,9 +51,31 @@ export default compose(
     handleSubmit: (values: FormState, { props, setErrors, resetForm, setSubmitting }) => {
       console.log(values)
       console.log(props)
-      props.createCollectionAction(values, () => {
-        setSubmitting(false)
-      })
+      if (props.data) {
+        // BACKEND FIX sections.resources -> sections.resourcesId :(
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/CreateCollectionForm/index.js]: lol ✝️

File: packages/kauri-web/components/containers/Link/View.js [hunk 4]
```diff
@@ -38,9 +39,11 @@ class Link extends React.Component<LinkProps> {
     let url = this.props.as || this.props.href || this.props.children.props.href
     const slug = this.props.toSlug ? slugify(this.props.toSlug, { lower: true }) : null;
     if (slug) url += `/${slug}`;
+    const { fullWidth = true } = this.props
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/Link/View.js]: Doesn't this affect existing designs?

File: packages/kauri-components/components/Button/PrimaryButton.re (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/Button/PrimaryButton.re]: Nah it's a transparent binding for the Primary button.js

File: packages/kauri-web/components/containers/Link/View.js (hunk 4 above)
Comment (rej156) [reply packages/kauri-web/components/containers/Link/View.js]: Doesn't cause it defaults to true

**Pull request merged**